.github/workflows/ci.yml

```diff
@@ -1199,6 +1199,96 @@ jobs:
         timeout-minutes: 5
         run: cat /tmp/flv_sc.log
 
+  mirroring_smoke_test_e2e:
+    runs-on: ${{ matrix.os }}
+    needs:
+      - build_image
+      - config
+    strategy:
+      matrix:
+        os: [ubuntu-latest]
+        rust-target: [x86_64-unknown-linux-musl]
+    steps:
+      - uses: actions/checkout@v4
+      - name: Setup BATS
+        uses: mig4/setup-bats@v1
+        with:
+          bats-version: ${{ env.BATS_VERSION }}
+
+      # Download artifacts from development build
+      - name: Download artifact - fluvio
+        uses: actions/download-artifact@v4
+        with:
+          name: fluvio-${{ matrix.rust-target }}
+          path: ~/bin
+      - name: Download artifact - fluvio-run
+        uses: actions/download-artifact@v4
+        with:
+          name: fluvio-run-${{ matrix.rust-target }}
+          path: ~/extensions
+      - name: Download artifact - fluvio-test
+        uses: actions/download-artifact@v4
+        with:
+          name: fluvio-test-${{ matrix.rust-target }}
+          path: ~/extensions
+      - name: Download Docker Image as Artifact
+        uses: actions/download-artifact@v4
+        with:
+          name: infinyon-fluvio-${{ matrix.rust-target }}.tar
+          path: /tmp
+
+      - name: Set up Fluvio binaries
+        run: |
+          chmod +x ~/bin/fluvio
+          chmod +x ~/extensions/fluvio-run
+          chmod +x ~/extensions/fluvio-test
+          mkdir -p ~/.fluvio/bin
+          mkdir -p ~/.fluvio/extensions
+          mv ~/bin/fluvio ~/.fluvio/bin/fluvio
+          mv ~/extensions/fluvio-run ~/.fluvio/bin/fluvio-run
+          mv ~/extensions/fluvio-test ~/.fluvio/bin/fluvio-test
+          echo "~/.fluvio/bin" >> $GITHUB_PATH
+
+      - name: Print version
+        run: fluvio version && fluvio-test -h
+
+      - name: Load Fluvio Docker Image
+        run: |
+          ls -la /tmp
+          docker image load --input /tmp/infinyon-fluvio-${{ matrix.rust-target }}.tar
+          docker image tag infinyon/fluvio:${{ github.sha }}-${{ matrix.rust-target }} infinyon/fluvio:${{ github.sha }}
+          docker image ls -a
+
+      - name: Export Docker Image to tarball
+        run: docker image save infinyon/fluvio:${{ github.sha }} --output /tmp/infinyon-fluvio.tar
+      - name: Install K3d
+        run: |
+          curl -s https://raw.githubusercontent.com/rancher/k3d/main/install.sh | TAG=${{ env.K3D_VERSION }} bash
+          ./k8-util/cluster/reset-k3d.sh
+      - name: Load image to K3d
+        run: |
+          k3d image import -k /tmp/infinyon-fluvio.tar -c fluvio
+
+      - name: Create k8 namespace
+        run: |
+          kubectl create ns fluvio-test
+
+      - name: Start cluster locally
```

Comment:
Should name this as `start edge cluster`.
Also, there should be at least 2 edge cluster

Reply:
Indeed I will change it.
About the edges, it seems that fluvio cluster gets SC and SPU by process, so I think it's impossible to have two clusters running in an environment. (Even with -skip-check option).

Reply:
Should be able to have to local cluster.  They just need to use different ports.  Try to to use env variable to override SC and SPU port

Reply:
Based on my tests, Fluvio uses too much const variables for local clusters, I don't think it's only SC and SPU ports the problem. 

I think we need to have these const vars changeable by cluster arguments:
LOCAL_PROFILE or implement skip_profile_creation for local clusters (works for k8s).
LOCAL_CONFIG_PATH
DEFAULT_DATA_DIR
LOCAL_SC_ADDRESS
LOCAL_SC_PORT
SC_PUBLIC_PORT
SC_PRIVATE_PORT

I would like to sync about it this week if this is a priority right now.

Reply:
There are alternatives
- Use K8 but with different name space. 
- Add option to cluster start with option to specify different ports and different SPU.

Reply:
using two edge nodes now